docker/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+
+ # SPDX-License-Identifier: Apache-2.0
+ # Licensed to the Ed-Fi Alliance under one or more agreements.
+ # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+ # See the LICENSE and NOTICES files in the project root for more information.
+
+
+version: "3"
+
+services:
+  amt-service:
+    container_name: amt-container
+    restart: always
+    build: .
+    links:
+      - mssql-service
+      - pg-service
```

Comment:
You don't need this section. Unless you are trying to run the tests completely inside of Docker? To be honest, that didn't occur to me. I was assuming  that the end-user is not _developing inside of Docker_. That they compiling on their own computer and then just need database instances to connect to.